extensions/type-snob/src/index.tsx

```diff
@@ -0,0 +1,93 @@
+const characters: Character[] = [
+  {
+    label: "Em Dash",
+    value: "—",
+    description: `> Em dashes can be used where parentheses might otherwise be used. They can also be used in place of a colon or when a comma is too weak but a period is too strong. An em dash is also used to show attribution of a quote, placed in front of the person’s name.
+
+— [typewolf.com/cheatsheet](https://www.typewolf.com/cheatsheet)`,
+  },
+  {
+    label: "En Dash",
+    value: "–",
+    description: `> An unspaced en dash can be used in place of the word “through” when indicating a range of values, such as numbers, times and dates. For example, July 5th–9th. It’s also used to show relationships between two things, such as a Boston–London flight.
+
+— [typewolf.com/cheatsheet](https://www.typewolf.com/cheatsheet)`,
+  },
+  {
+    label: "Figure Dash",
+    value: "‒",
+    description: `> The figure dash is the same width as a digit and is used for alignment purposes when setting numbers. It shouldn’t be used to indicate a range of numbers—that is the job of the en dash—instead, it should be used for setting things such as phone numbers (555‒867‒5309). Not all fonts contain figure dashes.
+
+— [typewolf.com/cheatsheet](https://www.typewolf.com/cheatsheet)`,
+    keywords: ["Number", "Dash"],
+  },
+  { label: "Minus", value: "−", keywords: ["Minus"] },
+  { label: "Opening Double Quote", value: "“", keywords: ["Left"] },
+  { label: "Closing Double Quote", value: "”", keywords: ["Right"] },
+  {
+    label: "Opening Single Quote",
+    value: "‘",
+    keywords: ["Left"],
+    description: singleQuoteDescription,
+  },
+  {
+    label: "Closing Single Quote & Apostrophe",
+    value: "’",
+    keywords: ["Apostrophe", "Right"],
+    description: singleQuoteDescription,
+  },
+  { label: "Prime (Feet)", value: "′", keywords: ["Feet"] },
+  { label: "Double Prime (Inches)", value: "″", keywords: ["Inches"] },
+  { label: "Ellipsis", value: "…" },
+  { label: "Left arrow", value: "←" },
+  { label: "Right arrow", value: "→" },
+  { label: "Up arrow", value: "↑" },
+  { label: "Down arrow", value: "↓" },
+  { label: "Multiplication", value: "×", keywords: ["Multiply", "Times"] },
+  { label: "Division", value: "÷", keywords: ["Divide"] },
+  { label: "Plus/Minus", value: "±" },
+  { label: "Less Than or Equal To", value: "≤" },
+  { label: "Greater Than or Equal To", value: "≥" },
+  { label: "Not Equal To", value: "≠" },
+  { label: "Almost equal", value: "≈" },
+  { label: "Cents", value: "¢" },
+  { label: "Degree", value: "°" },
+  { label: "Registered", value: "®" },
+  { label: "Copyright", value: "©" },
+  { label: "Trademark", value: "™" },
+];
+
+export default function Command() {
+  return (
+    <List isShowingDetail={false}>
+      {characters.map((char) => (
+        <List.Item
+          key={char.label}
+          title={char.value}
+          // detail={<List.Item.Detail markdown={`# ${char.label}\n${char.description ?? ""}`} />}
```

Comment:
Wanna remove this comment, or planing on implementing it? 

We can then also remove `isShowingDetail={false}`

Reply:
Planning to implement in a subsequent version, but I've pushed an update to remove it for now: https://github.com/raycast/extensions/pull/5544/commits/87b41e431719096fad3941120547faaa9c6b2a94